cmd/get/task.go

```diff
@@ -17,20 +18,26 @@ const (
 	taskShort = "Gets task resources"
 	taskLong  = `
 Retrieves all the task within project and domain.(task,tasks can be used interchangeably in these commands)
+
 ::
 
  bin/flytectl get task -p flytesnacks -d development
 
-Retrieves task by name within project and domain.
 
+Retrieves task by name within project and domain.
 ::
 
  bin/flytectl task -p flytesnacks -d development core.basic.lp.greet
```

Comment:
nit: these examples have "lp" in the name ... seems a bit confusing in the context of tasks